Dev_Document.py

```diff
@@ -201,4 +201,4 @@ def waitPag_State(self, state):
             self.waitPage.hide()
         elif state == CMD_DOWNLOAD_OK:
             self.waitPage.hide()
-            QMessageBox.information(self,"温馨提示","下载成功！")
+            QMessageBox.information(self,"Warning","download successful!")
```

Comment:
"温馨提示" can be translated to "Hint" or "Note" in English.
```python
QMessageBox.information(self,"Note","Download successfully!")
```

Reply:
I actually dropped the development of mesh project with these modules, because it is impossible to get any support.
I was hoping that someone could take this TBxx modules and make some good tutorial in English, so that the rest of us could follow.
BR.